Mathlib/FieldTheory/Normal.lean

```diff
@@ -355,10 +355,55 @@ theorem AlgEquiv.restrict_liftNormal (χ : K₁ ≃ₐ[F] K₁) [Normal F K₁]
     (algebraMap K₁ E).injective
       (Eq.trans (AlgEquiv.restrictNormal_commutes _ K₁ x) (χ.liftNormal_commutes E x))
 
+/-- The group homomorphism given by restricting an algebra isomorphism to a normal subfield
+is surjective. -/
 theorem AlgEquiv.restrictNormalHom_surjective [Normal F K₁] [Normal F E] :
     Function.Surjective (AlgEquiv.restrictNormalHom K₁ : (E ≃ₐ[F] E) → K₁ ≃ₐ[F] K₁) := fun χ =>
   ⟨χ.liftNormal E, χ.restrict_liftNormal E⟩
 
+/-- The group homomorphism given by restricting an algebra isomorphism to itself
+is the identity map. -/
+@[simp]
+theorem AlgEquiv.restrictNormalHom_id (F K : Type*)
+    [Field F] [Field K] [Algebra F K] [Normal F K] :
+    AlgEquiv.restrictNormalHom (F := F) (K₁ := K) K = MonoidHom.id (K ≃ₐ[F] K) := by
```

Comment:
Do you see a reason why all the variables are implicit in `AlgEquiv.restrictNormalHom`? Maybe you can change them in this PR if this does not cause too much troubles.

Reply:
I think not all, the intermediate field being restict to is explicit, and for all the places used in this PR, only one of the three  things like (F := F) (K1 := K) in `AlgEquiv.restrictNormalHom_comp` is need to be written explicitly because it involve to much `AlgEquiv.restrictNormalHom`.